clients/ui/frontend/src/__tests__/cypress/cypress/utils/testConfig.ts

```diff
@@ -0,0 +1,19 @@
+import path from 'path';
+import { env } from 'process';
+import dotenv from 'dotenv';
+
+[
+  `.env.cypress${env.CY_MOCK ? '.mock' : ''}.local`,
+  `.env.cypress${env.CY_MOCK ? '.mock' : ''}`,
+  '.env.local',
+  '.env',
+].forEach((file) =>
+  dotenv.config({
+    path: path.resolve(__dirname, '../../../../../', file),
+  }),
+);
+
+export const BASE_URL = env.BASE_URL || '';
```

Comment:
Ok great, we'll need to do follow ups with env variables, but head to a great start here.